modules/oxide/src/less/theme/components/toolbar-button/toolbar-number-input.less

```diff
@@ -0,0 +1,53 @@
+//
+// Toolbar number input
+//
+
+@toolbar-number-input-button-height: 28px;
+@toolbar-number-input-button-width: 24px;
+@toolbar-number-input-input-wrapper-width: 75px;
```

Comment:
It seems too wide for the default skin
![image](https://user-images.githubusercontent.com/44773772/207896404-cc25e724-99cd-4fdd-aaa7-47faa8c98c7f.png)

Reply:
Also it doesn't look like the designs didn't that have a gray background for the input? Maybe double check with Paul.

Reply:
ok, I did a check with Paul, now it should be ok